src/themes/darkTheme.ts

```diff
@@ -52,6 +53,11 @@ const darkTheme = createMuiTheme({
       },
     },
   },
+  props: {
+    MuiTypography: {
+      color: 'textPrimary',
```

Comment:
ups 
![bilde](https://user-images.githubusercontent.com/6313468/106742672-9f710100-661d-11eb-891a-c45fb1a87b83.png)

Reply:
hehe, liker at du finner dine egne feil

Reply:
syns det blir mest rett å bruke textPrimary her, men bruker bare primary som en quick fix i stedet for å slippe av labels blir usynlig. får ta en ny runde på dette senere